### DISCUSSION TITLE
First api request take long time

### DISCUSSION THREAD

**Discussion by lohnsonok** [prisma/prisma#7700] [Q&A]
First client connexion (login sample) take long time (Prisma connexion) to return result.
After the first response, all next request is fast 

How to increase first response speed ?

**Comment by ryands17** [comment]
@lohnsonok 👋 

The first API shouldn't take that long. 

1. In which location is the app hosted and where's the database? Are they in the same geographic location?
2. Are you using serverless or long-running servers like Heroku or EC2?

**Reply by lohnsonok** [reply]
I'm using heroku

**Reply by ryands17** [reply]
What about the database? Where is it hosted and is it in a different location?

**Reply by lohnsonok** [reply]
> What about the database? Where is it hosted and is it in a different location?

Here is the information about my application on heroku

Region          United States
Stack             heroku-20
Framework    Node.js
Slug size       107.1 MiB of 500 MiB

**Reply by ryands17** [reply]
One way to do this would be to call `prisma.$connect()` when your application just starts. So adding this at the root of the file should save some amount of time.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
